frontend/src/pages/projects/ProjectDetailsContext.tsx

```diff
@@ -140,7 +120,13 @@ const ProjectDetailsContextProvider: React.FC = () => {
         serverSecrets,
       }}
     >
-      <Outlet />
+      {featureFlagEnabled(dashboardConfig.spec.dashboardConfig.disablePipelines) ? (
+        <PipelineContextProvider namespace={project.metadata.name}>
+          <Outlet />
+        </PipelineContextProvider>
+      ) : (
+        <Outlet />
+      )}
```

Comment:
We might want to look at some params for Model Serving -- it's throwing all kinds of 404 errors since I do not have the stack installed (flag is off). cc @lucferbux